content/pt-br/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Aplicando os Padrões de Segurança do Pod, Configurando o Controlador de Admissão Embutido
```

Comment:
Thought: the original title is "Enforce Pod Security Standards by Configuring the Built-in Admission Controller". This connection between the two elements was lost in the translation.
Consider changing it to "Aplicando os Padrões de Segurança do Pod _Através_ da Configuração do Controlador de Admissão Embutido"